doc/images.md

```diff
@@ -13,29 +13,113 @@ flutter:
 
 It has to be a PNG file and it can have transparency.
 
+## Loading images
+
+Flame bundles an utility class called `Images` that allows you to easily load and cache images from the assets into the memory.
+
+Flutter has a collection of types related to images, and converting everything properly from a local asset to the Image that can be drawn on Canvas is a bit of a pain. This class allows you to obtain an Image that can be drawn on a Canvas using the `drawImageRect` method.
```

Comment:
I might have originally wrote this, but since we are talking about the type system, I think "a collection of types" might be just too confusing for now reason, let's say "a handful of types" instead :)

Reply:
let's also tone down a bit of a pain to a bit convoluted